lib/rules/selector-nested-pattern/index.js

```diff
@@ -39,20 +51,35 @@ const rule = (primary) => {
 				return;
 			}
 
-			const selector = ruleNode.selector;
+			if (splitList) {
+				ruleNode.selectors.forEach((selector) => {
+					if (!normalizedPattern.test(selector)) {
+						report({
+							result,
+							ruleName,
+							message: messages.expected,
+							messageArgs: [selector, primary],
+							node: ruleNode,
+							word: selector,
+						});
+					}
+				});
+			} else {
+				const selector = ruleNode.selector;
 
-			if (normalizedPattern.test(selector)) {
-				return;
-			}
+				if (normalizedPattern.test(selector)) {
+					return;
+				}
 
-			report({
-				result,
-				ruleName,
-				message: messages.expected,
-				messageArgs: [selector, primary],
-				node: ruleNode,
-				word: selector,
-			});
+				report({
+					result,
+					ruleName,
+					message: messages.expected,
+					messageArgs: [selector, primary],
+					node: ruleNode,
+					word: selector,
+				});
+			}
```

Comment:
[suggest] How about reducing code duplication like this?

```js
const selectors = splitList ? ruleNode.selectors : [ruleNode.selector];

for (const selector of selectors) {
	if (normalizedPattern.test(selector)) {
		continue;
	}

	report({ /*...*/ });
}
```

Reply:
@ybiquitous 

Thanks for your comment!
I updated the change.

https://github.com/stylelint/stylelint/pull/6896/commits/33239f984d6606fed515836f656ecda43a6fa93b